static/carousel/js/carousel.js

```diff
@@ -233,17 +241,37 @@
             }
         };
 
+        this.dynamicSort =function(property) {
```

Comment:
Sorting must happen in the backend, so you should listen to the option and pass it to the data request params:
n=200&status=approved&o=title&od=DESC

Reply:
:+1: